app/models/comment.rb

```diff
@@ -20,13 +20,11 @@ class Comment < ActiveRecord::Base
   private
 
   def create_notification
-    article = Article.find_by(self.article_id)
```

Comment:
would probably recommend not creating the notification if the commenter and article owner are the same person?